Resources/Prototypes/Recipes/Reactions/fun.yml

```
@@ -201,3 +201,29 @@
   effects:
     - !type:CreateEntityReactionEffect
       entity: MaterialGunpowder
+
+- type: reaction
+  id: SolidifyGold
+  impact: Low
+  quantized: true
+  reactants:
+    Gold:
+      amount: 10
+    Frostoil:
```

```suggestion
    FrostOil:
```
Time for a web edit :godo: